### ISSUE TITLE
NDBC GOM data 

### ISSUE THREAD

**Issue by kjschaudt** [ropensci/rnoaa#387]
Any one know what is going on?   (1) . According to www.ndbc.noaa.gov, the NDBC primary processing servers were shut off due to a facilities issue which has shut down the feed to the station data.  (2)  Further it appears that data from the Gulf of Mexico Oil and Gas platforms will be transferred from NDBC to GCOOS.  https://data.gcoos.org website, by the end of the month.   Data formats and archiving will differ.   Hard to evaluate impact since the NDBC data feed is down and the GCOOS site is spinning up.   

Will Rnoaa be able to handle the new data formats and sources?

**Comment by sckott** [comment]
Thanks for opening the issue @kjschaudt 

I do not know - is the answer. Did you notice this due to one or more rnoaa functions failing? If so, which ones? 

We'll very likely be able to handle it

**Comment by ecoflo** [comment]
Hi @sckott - hope you can help with the error I receive from RNOAA package. I am not sure this relates to the changes described above, when using the ghcnd_search function, it returns systematically: "Error: input not of correct format". Even when running the example function such as: 

`ghcnd_search("AGE00147704", var = "PRCP")`

Your advice / help would be greatly appreciated! 👍

**Comment by sckott** [comment]
@ecoflo please open a separate issue for that. I can not replicate the problem.

**Comment by sckott** [comment]
@kjschaudt What `rnoaa` functions use the data you're talking about above?

**Comment by kjschaudt** [comment]
I believe it was the buoy function for our application.  The NDBC data that were offline included the METAR aviation stations and the ADCP current meters.  Don't recall if I managed to get the buoy function fully operational or not.   I ended up scraping the website to get the job done.   A partial list of the ADCP stations that went offline at NDBC and are now coming up at GCOOS is listed below.  
42363 
42370
42374
42375
42377
42383
42388
42398
42399
42400
42851
42864
42865
42873
42876
42882
42884
42887
42888
42891
42896
42902
42906
